### PR TITLE
Remove Several Deprecated Commands

### PR DESCRIPTION
**Pull request by hacdias**
This PR removes several deprecated commands (see code). Doing this for the sake of maintainability. Most of this commands have been deprecated for over a year (and some for almost 5 years).

### CODE REVIEW

**Comment by Jorropo** [comment]
Does that means we can remove things from boxo too ? Stuff like urlstore, ...

**Comment by hacdias** [comment]
@Jorropo the URLStore is still a thing (experiment) and exists, but is used with the `ipfs add` command. This just removes the deprecated command.

File: docs/changelogs/v0.26.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+### 🔦 Highlights
+
+#### Several deprecated commands have been removed
+
+Several deprecated commands have been removed:
+
+- `ipfs urlstore` deprecated in [April 2019, Kubo 0.4.21](https://github.com/ipfs/kubo/commit/8beaee63b3fa634c59b85179286ad3873921a535)
+- `ipfs repo fsck` deprecated in [July 2019, Kubo 0.5.0](https://github.com/ipfs/kubo/commit/288a83ce7dcbf4a2498e06e4a95245bbb5e30f45)
+- `ipfs file` (and `ipfs file ls`) deprecated in [November 2020, Kubo  0.8.0](https://github.com/ipfs/kubo/commit/ec64dc5c396e7114590e15909384fabce0035482)
+- `ipfs dns` deprecated in [April 2022, Kubo 0.13](https://github.com/ipfs/kubo/commit/76ae33a9f3f9abd166d1f6f23d6a8a0511510e3c)
```
Comment (lidel) [inline docs/changelogs/v0.26.md:24]:
```suggestion
- `ipfs dns` deprecated in [April 2022, Kubo 0.13](https://github.com/ipfs/kubo/commit/76ae33a9f3f9abd166d1f6f23d6a8a0511510e3c), use `ipfs resolve /ipns/{name}` instead.
```

File: docs/changelogs/v0.26.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+### 🔦 Highlights
+
+#### Several deprecated commands have been removed
+
+Several deprecated commands have been removed:
+
+- `ipfs urlstore` deprecated in [April 2019, Kubo 0.4.21](https://github.com/ipfs/kubo/commit/8beaee63b3fa634c59b85179286ad3873921a535)
+- `ipfs repo fsck` deprecated in [July 2019, Kubo 0.5.0](https://github.com/ipfs/kubo/commit/288a83ce7dcbf4a2498e06e4a95245bbb5e30f45)
+- `ipfs file` (and `ipfs file ls`) deprecated in [November 2020, Kubo  0.8.0](https://github.com/ipfs/kubo/commit/ec64dc5c396e7114590e15909384fabce0035482)
```
Comment (lidel) [inline docs/changelogs/v0.26.md:23]:
```suggestion
- `ipfs file` (and `ipfs file ls`) deprecated in [November 2020, Kubo  0.8.0](https://github.com/ipfs/kubo/commit/ec64dc5c396e7114590e15909384fabce0035482), use `ipfs ls` and `ipfs files ls` instead.
```

File: docs/changelogs/v0.26.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+### Overview
+
+### 🔦 Highlights
+
+#### Several deprecated commands have been removed
+
+Several deprecated commands have been removed:
+
+- `ipfs urlstore` deprecated in [April 2019, Kubo 0.4.21](https://github.com/ipfs/kubo/commit/8beaee63b3fa634c59b85179286ad3873921a535)
```
Comment (lidel) [inline docs/changelogs/v0.26.md:21]: :warning:  @hacdias what is the alternative to `ipfs urlstore add` ?
I know for filestore we can use `ipfs add --nocopy`, but does the same apply here? If so, mind updating  https://github.com/ipfs/kubo/blob/master/docs/experimental-features.md#ipfs-urlstore too?

If not, we need to keep this.

```suggestion
- `ipfs urlstore` deprecated in [April 2019, Kubo 0.4.21](https://github.com/ipfs/kubo/commit/8beaee63b3fa634c59b85179286ad3873921a535), use {TODO} instead
```

**Review by lidel — CHANGES_REQUESTED** [review]

File: docs/changelogs/v0.26.md (hunk 3 above)
Comment (hacdias) [reply docs/changelogs/v0.26.md:21]: Yes, it does exactly the same. It's documented. I will update.

File: docs/changelogs/v0.26.md [hunk 4]
```diff
@@ -18,10 +18,10 @@
 
 Several deprecated commands have been removed:
 
-- `ipfs urlstore` deprecated in [April 2019, Kubo 0.4.21](https://github.com/ipfs/kubo/commit/8beaee63b3fa634c59b85179286ad3873921a535)
+- `ipfs urlstore` deprecated in [April 2019, Kubo 0.4.21](https://github.com/ipfs/kubo/commit/8beaee63b3fa634c59b85179286ad3873921a535), use `ipfs add -q --nocopy --cid-version=1 {file}` instead.
```
Comment (lidel) [inline docs/changelogs/v0.26.md:21]: nit: should this be `{url}` instead of `{file}` ?  
(urlstore != filestore, these are separate experiments)

**Review by lidel — APPROVED** [review]

**Comment by hacdias** [comment]
I will merge the PR once CI is green. It won't be cherry-picked for 0.25 - this will only land for 0.26.

**Pull request merged**